### ISSUE TITLE
Feature request: speed in map units per tic

### ISSUE THREAD

**Issue by MaartenCL** [borogk/cameraman#4]
Speed can now be determined by distance or time. I intend to chain some recordings, and therefore it would be handier if I could indicate how fast the camera is supposed to go in terms of map units per tic, so I'd like to propose a third speed mode. In this case a speed setting of 1 would indicate that it would take the camera 100 tics to move 100 map units.

You think this would be possible to implement?

Even better would be if it would be possible to indicate the speed at the start and the end of the path, thus making the camera accelerate or decelerate. But that would require a second speed parameter.

(I could try and do it myself, but I'm a web developer and this is a whole different ballpark. Would take me a while to be able to compile this in the first place.)

**Comment by MaartenCL** [comment]
Indeed, acceleration should be a separate feature request, so let's leave that for now. And indeed I was hoping to input speed for angle and bezier paths the same way as for linear.

I should at least try to compile the project myself, especially since you're not working on this at the moment anyway. If I succeed, I could do a pull request. Others might be interested in that as well.

**Comment by borogk** [comment]
Closed after long period of inactivity
